TrackingTools/TrackAssociator/python/DetIdAssociatorESProducer_cff.py

```diff
@@ -36,12 +36,18 @@
     etaBinSize = cms.double(0.125),
     nEta = cms.int32(48),
     nPhi = cms.int32(48),
-    includeBadChambers = cms.bool(False)
+    includeBadChambers = cms.bool(False),
+    includeGEM = cms.bool(False),
+    includeME0 = cms.bool(False)
 )
 
 # If running in Run 2, include bad chambers
 eras.run2_common.toModify( muonDetIdAssociator, includeBadChambers = True )
 
+# include GEM & ME0 for phase2
+eras.phase2_muon.toModify( muonDetIdAssociator, includeGEM = True )
```

Comment:
this should be also in run3_GEM (if you add to run3_GEM, the change in phase2_muon is not necessary)